src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,20 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int bladeWidth;
```

Comment:
The field 'bladeWidth' should have an access modifier to encapsulate the field properly, typically 'private' to make sure it is not accessible directly from outside the class.